cmd/minishift/cmd/profile/profile.go

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright (C) 2017 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package profile
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var ProfileCmd = &cobra.Command{
+	Use:     "profile SUBCOMMAND [flags]",
+	Aliases: []string{"instance"},
+	Short:   "Manages Minishift profiles.",
+	Long:    "Manages Minishift profile. Use the sub-commands to list, set profile",
```

Comment:
"Manages Minishift profile. Use the sub-commands to list, set profile" -> Allows you to create and manage multiple Minishift instances (profiles). Use the sub-commands to active and list existing profiles."